src/core/SampleBuffer.cpp

```diff
@@ -1425,7 +1425,7 @@ QString SampleBuffer::tryToMakeRelative( const QString & file )
 		// Iterate over all valid "data:/" searchPaths
 		for ( const QString & path : QDir::searchPaths( "data" ) )
 		{
-			QString samplesPath = QString( path + samplesSuffix ).replace( QDir::separator(), '/' );
+			QString samplesPath = QDir::cleanPath( path + samplesSuffix ) + "/";
```

Comment:
I believe `ensureTrailingSlash` may be better here.  The previous logic doesn't add any slashes.

Reply:
Maybe, but don't have to be. `QDir::cleanPath` always strips trailing slash and previous logic always gives a path with trailing slash.
BTW, `ensureTrailingSlash` is defined in `ConfigManager.cpp`. If you want to use that function, you should move that into an header first.